.github/workflows/label.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler
+
+name: Labeler
+on: [pull_request]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+    - uses: actions/labeler@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        sync-labels: true
```

Comment:
Will this mean labels we add manually would be removed, e.g. if it's not quite right? (I'm guessing it won't always be)

Reply:
The docs say "Whether or not to remove labels when matching files are reverted or no longer changed by the PR."

Not clear if it removes tags added manually, or only removes tags that were added previously by the action. e.g. does it look at individual commits and looks for reverted changes? 🤷🏽